ReSwift/CoreTypes/MainStore.swift

```diff
@@ -70,11 +68,10 @@ public class MainStore<State: StateType>: Store {
         }
 
         isDispatching = true
-        let newState = self.reducer._handleAction(self.state, action: action)
+        let newState = reducer._handleAction(action, state: state)
         isDispatching = false
 
-        self.state = newState as! State
-
+        state = newState as! State
         return action
```

Comment:
I've *tried* to be consistent, and keeping an empty line before `return` statements that actually return a value. I think that makes the code easier to parse.